assets/js/modules/analytics/dashboard/dashboard-widget-overview.js

```diff
@@ -70,11 +101,13 @@ class AnalyticsDashboardWidgetOverview extends Component {
 			averageSessionDurationChange,
 		} = overviewData;
 
+		const directTotalUsersData = directTotalUsers && directTotalUsers[ 0 ].data.totals[ 0 ].values[ 0 ];
```

Comment:
I'm a bit weary of accessing a path so long without knowing for sure that all of the segments exist.

Let's use `get` from lodash to access it safely, just in case.

Reply:
Good point. I wasn't aware of `get` are we using that elsewhere already? What about using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)?

Reply:
Yeah, we have used `get` in a few places (e.g. [Tag Manager setup](https://github.com/google/site-kit-wp/blob/58891192721c60496c30944fb5dc37b591c9bccd/assets/js/modules/tagmanager/setup.js#L172-L173)). I like the idea of using optional chaining but that requires the addition of [a babel plugin](https://babeljs.io/docs/en/next/babel-plugin-proposal-optional-chaining) which should probably be done in its own PR.

Reply:
Switched to `lodash.get` in https://github.com/google/site-kit-wp/pull/1277/commits/8a0d6ca403f0ce7082d282111ec97e975af329bd.